docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,49 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. These tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+In this page, we'll discuss how Ubuntu Pro tokens work and how machines are counted against your subscription.
+
+Ubuntu Pro token(s)
+--------------------
+
+- Obtaining Pro tokens:
+     After subscribing to Ubuntu Pro, you can access your subscriptions through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each subscription includes a Ubuntu Pro token that can be used to attach as many machines as the subscription allows.
+
+- Attaching Pro tokens:
+    Command: Use ``sudo pro attach`` to connect a machine to your Ubuntu Pro subscription.
```

Comment:
```suggestion
    You can use the ``sudo pro attach`` command to connect a machine to your Ubuntu Pro subscription.
```

Reply:
Having this after the "obtain a token" point makes it appear like this is a list of tasks that they need to do in order, but they don't need the token *a priori* to be able to attach. They only need the subscription - so maybe worth adding a line here to point out that the attach process allows them to choose the token they want to use?

Reply:
The attach basically either allows them to use their obtained token to attach or use `magic attach` which directs them to the pro dashboard where they use pro attach and the command takes them to the pro website. We can clarify this in the sentence.